apps/root/src/common/utils/errors.tsx

```diff
@@ -60,3 +64,25 @@ export const shouldTrackError = (error?: { code?: string; message?: string; reas
 
   return true;
 };
+
+const getTransactionExecutionErrorCode = (error: TransactionExecutionErrorType) => {
```

Comment:
Carefull that not all errors are going to be caused by viem, we could have an error on our side, we should not treat every error as a viem one, but It might be one

Reply:
I undestand. I've made some changes in order to make this more flexible